addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1073,6 +1084,37 @@ var SnippetsMenu = Widget.extend({
             }
         });
     },
+    /**
+     * Adds an entry in the left panel box which holds the invisible elements.
+     * The entry will contains an 'Edit' button to activate the snippet.
+     *
+     * @param {jQuery} $invisibleSnippet
+     *        The snippet for which an entry need to be added in the left panel.
+     */
+    _addInvisibleElementsToLeftPanel: function ($invisibleSnippet) {
+        const self = this;
+        $invisibleSnippet.each(function () {
+            const invisEntryId = _.uniqueId('invisEntry');
+            const $snippet = $(this);
+            $snippet.attr('invisiblePanelId', invisEntryId);
+
+            const $invisEntry = $('<p/>', {
+                id: invisEntryId,
+            }).text($snippet.attr('string'));
+            $invisEntry.append($('<a/>', {
+                href: '#',
+                text: _t("Edit"),
+                class: 'ml-2',
+            }));
+            $invisEntry.on('click', (ev) => {
+                ev.preventDefault();
+                self._activateSnippet($snippet).then(() => {
+                    $snippet.trigger('snippet_activated_from_invis');
+                });
+            });
+            $('.o_we_invisible_el_panel').append($invisEntry);
```

Comment:
Should not you use you pretty variable?

Reply:
Whoops yes my bad, used `$(this.invisibleDOMPanelEl).append($invisEntry);`